catalog/spatial/registry/registry-rest-endpoint/pom.xml

```diff
@@ -50,11 +50,7 @@
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
-            <version>3.4</version>
-        </dependency>
-        <dependency>
-            <groupId>org.codice.ddf.registry</groupId>
-            <artifactId>registry-schema-bindings</artifactId>
+            <version>3.0</version>
```

Comment:
I ended up creating https://codice.atlassian.net/browse/DDF-2223 to capture the plan for commons-lang and I would expect that to be a separate PR across multiple modules.